### PR TITLE
Update Documentation and Processes

### PR DESCRIPTION
**Pull request by joebell93**
Updates before 1.0

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,30 +2,38 @@
...
-
-## Pull requests
+## Pull Requests
 
 0. Create a new local branch for your work.
 0. As early as possible, create a pull request in the [appropriate repository](https://github.com/sky-uk/toolkit#structure). Make sure you give enough information in the pull request description, and add the label `in progress` with any other appropriate label.
 0. Once any conflicts have been fixed and you're ready for your code to be reviewed, remove the `in progress` label and add `hunting for squirrels`. 
-0. Get a code review by one or two developers. One of these **must** be a [core maintainer of Toolkit](https://github.com/sky-uk/toolkit#maintainers). 
+0. Get a code review by two developers. One of these **must** be a [core maintainer of Toolkit](https://github.com/sky-uk/toolkit#maintainers). 
 	- You need two :shipit: :shipit: left as comments on the pull request for when it's an addition or beaking change.
```
Comment (skitson) [inline CONTRIBUTING.md]: Not part of your changes but `beaking` should be `breaking`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,30 +2,38 @@
 
 We greatly appreciate and encourage people contributing back into the Toolkit!
 
-## Creating issues
+## Creating Issues
 
 Submit issues to the issue tracker on the [appropriate repository](https://github.com/sky-uk/toolkit#structure) for suggestions, recommendations, and bugs.
 
-When submitting issues, please make sure you include the following:
+When submitting issues, please make sure you follow the template provided.
```
Comment (welikeideas) [inline CONTRIBUTING.md]: Could we make this a little clearer? Does this need to reference a url for the template?

File: CONTRIBUTING.md (hunk 1 above)
Comment (joebell93) [inline CONTRIBUTING.md]: image

File: CONTRIBUTING.md (hunk 2 above)
Comment (joebell93) [inline CONTRIBUTING.md]: @welikeideas so when you create an issue on toolkit-core or toolkit-ui, it automatically provides a template 😃 

e.g. https://github.com/sky-uk/toolkit-core/issues/new

Just noticed this isn't on the main repo though, so good spot

File: CONTRIBUTING.md (hunk 1 above)
Comment (joebell93) [inline CONTRIBUTING.md]: (Good spot)

**Pull request merged**